modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/account_contact/edit_address.jsp

```diff
@@ -0,0 +1,208 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+EditContactInformationDisplayContext editContactInformationDisplayContext = new EditContactInformationDisplayContext("address", request, renderResponse);
+
+editContactInformationDisplayContext.setPortletDisplay(portletDisplay, portletName);
+
+Address address = null;
+
+long countryId = 0L;
+long regionId = 0L;
+
+if (editContactInformationDisplayContext.getPrimaryKey() > 0) {
+	address = AddressServiceUtil.getAddress(editContactInformationDisplayContext.getPrimaryKey());
+
+	countryId = address.getCountryId();
+	regionId = address.getRegionId();
+}
+
+PortalUtil.addPortletBreadcrumbEntry(request, LanguageUtil.get(request, "addresses"), editContactInformationDisplayContext.getRedirect());
+
```

Comment:
Based on this https://github.com/liferay/liferay-portal/commit/2445f2b8ba8b3eb037885e4d36cef3417fc1dc8d this line should be removed